### PR TITLE
nerf-api: Install wintun driver for windows

### PR DESCRIPTION
**Pull request by NerijusRazvodovskis**
Going to use `Wintun` driver since it's easier to set up and we don't need to edit any registries like with `TUN/TAP`.

Also `nebula` tries to search firstly `Wintun` and fallbacks to `TUN/TAP` later if it's not found.

### CODE REVIEW

**Comment by NerijusRazvodovskis** [comment]
Will  try to do with flag better

File: nebula_darwin.go [hunk 1]
```diff
@@ -26,6 +26,18 @@ func NebulaExecutable() string {
 	return path.Join(NebulaDir(), "nebula")
 }
 
+func wintunDownloadLink() string {
```
Comment (ton31337) [inline nebula_darwin.go:29]: Why do we need these defined here at all? Just bundle everything in a single method and that's it in `WintunDownload()`.

Comment (NerijusRazvodovskis) [reply nebula_darwin.go:29]: true, thanks 🙇🏻

File: nebula_common.go [hunk 2]
```diff
@@ -267,3 +268,61 @@ func NebulaDownload() (err error) {
 
 	return nil
 }
+
+// WintunDownload used to download Wintun driver
+func WintunDownload() (err error) {
+
+	wintunDownloadLink := "https://github.com/hostinger/packages/releases/download/v1.0.0/wintun-0.14.1.zip"
```
Comment (ton31337) [inline nebula_common.go:275]: nit: This (and other URLs) would be handy if could be specified in CLI (as arguments).

**Review by ton31337 — APPROVED** [review]

File: nebula_common.go (hunk 2 above)
Comment (NerijusRazvodovskis) [reply nebula_common.go:275]: Noted, will do it soon

**Pull request merged**